frontend/sass/evictionfree/styles.scss

```diff
@@ -79,7 +79,6 @@ $max-content-width: 1440px;
   text-transform: uppercase;
   letter-spacing: 2px;
   font-weight: 200;
-  margin: 0 2rem 2rem 0;
```

Comment:
Are these text styles also relevant to reverting, or is this new?

Reply:
yes, so these were part of reverting some css changes that were specific to the "This tool is suspended" homepage.